api/users/tests/tests_login.py

```diff
@@ -0,0 +1,125 @@
+import json
+import responses
+import re
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from contextlib import contextmanager
+
+from rest_framework.reverse import reverse
+from rest_framework.status import HTTP_200_OK, HTTP_201_CREATED, HTTP_401_UNAUTHORIZED, HTTP_404_NOT_FOUND, HTTP_400_BAD_REQUEST
```

Comment:
Hello @IssaIan and @a-braham ,
Instead of doing an import of the many statuses, you can just inmport status from rest framework then do status.your-required-status

Reply:
Done. Thanks.